File: WindowsCE/NurSample_vs2008_VB/NurSampleCE_VB/NXP.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Text
Imports System.Windows.Forms

Imports NurApiDotNet
'NurApi wrapper
Partial Public Class NXP
    Inherits UserControl
    Public Sub New()
        InitializeComponent()
    End Sub

    ''' <summary>
    ''' Sets the hadle of nur API.
    ''' </summary>
    ''' <param name="hNur">Handle of NurApi.</param>
    Public Sub SetNurApi(ByVal hNur As NurApi)
        Me.Enabled = True
        nxpProductStatusFlag.SetNurApi(hNur)
        nxpEasAlarm.SetNurApi(hNur)
        nxpReadProtect.SetNurApi(hNur)
    End Sub
End Class

File: NurSamplePC_vs2008_CS/NurSamplePC/NXP.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Windows.Forms;

using NurApiDotNet; //NurApi wrapper

namespace NurSample
{
    public partial class NXP : UserControl
    {
        public NXP()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Sets the hadle of nur API.
        /// </summary>
        /// <param name="hNur">Handle of NurApi.</param>
        public void SetNurApi(NurApi hNur)
        {
            this.Enabled = true;
            nxpProductStatusFlag.SetNurApi(hNur);
            nxpEasAlarm.SetNurApi(hNur);
            nxpReadProtect.SetNurApi(hNur);
        }
    }
}
